lib/internal/mime.js

```diff
@@ -144,24 +251,33 @@ class MIMEParams {
     const data = this.#data;
     name = `${name}`;
     value = `${value}`;
```

Comment:
why though?
can value and name be not a string?
maybe creating a string only if it is not a string is faster.

```suggestion
    typeof name !== 'string' && (name = `${name}`);
    typeof value !== 'string'&& (value = `${value}`);
```

Reply:
Not sure, but I believe that it might be to trigger the `toString` function

Reply:
but why would you need that on a string?

Reply:
Imagine that is in case the value passed isn't a `string` so we enforce the primitive representation or whatever `toString` returns.

Reply:
> maybe creating a string only if it is not a string is faster.

I doubt it, V8 should already have it optimized, I would expect doing the test manually as suggested above would be slower than not doing it.